deploy/deployment.yaml

```diff
@@ -17,22 +17,17 @@ spec:
       - name: rbac-query-proxy
         image: blue0/rbac-query-proxy:latest
```

Comment:
I saw the main branch is using our official image. why it is your private image without any changes?

Reply:
my local repo did not accept the latest commits, will close this one and create a new pr